golem/resources/invoice/invoice.py

```diff
@@ -83,17 +83,24 @@ async def validate_and_accept(self, allocation: Allocation) -> None:
             ) = await self.get_time_and_amount_since_latest_debit_notes(
                 invoice_data.timestamp, amount_due
             )
+            coeffs = LinearCoeffs.from_properties(agreement_data.properties)
+            infrastructure = InfrastructureProps.from_properties(agreement_data.properties)
 
             max_cost, max_cost_since_latest_debit_notes = validate_payment_max_cost(
-                coeffs=LinearCoeffs.from_properties(agreement_data.properties),
-                inf=InfrastructureProps.from_properties(agreement_data.properties),
+                coeffs=coeffs,
+                inf=infrastructure,
                 duration=agreement_data.agreement_duration,
                 amount=amount_due,
                 time_since_last_debit_note=cumulative_time_since_last_dn,
                 amount_since_last_debit_note=cumulative_amount_since_last_dn,
             )
         except PaymentValidationException:
-            logger.warning(f"Invoice {self.id} validation failed", exc_info=True)
+            logger.warning(
```

Comment:
shouldn't the additional information be logged with `.debug` ?

Reply:
I would prefer to have them easier accessible for sake of our mainet bot until we have confidence that validation works as intended in most edge cases.